COVIDWatch iOS/ViewControllers/Share.swift

```diff
@@ -27,24 +27,25 @@ class Share: BaseViewController {
         img!.frame.size.height = 259
 
         img!.center.x = view.center.x - 5
-        img!.center.y = 240
+        img!.center.y = 200
         if screenHeight <= 736.0 { img!.center.y *= scalingFactor! }
+        if screenHeight <= 812.0 { img!.center.y += 40 }
```

Comment:
Can this be updated to not force unwrap?

Reply:
100%, I missed that, I'll make the change